server/lib/src/idm/applinks.rs

```diff
@@ -51,11 +51,13 @@ impl<'a> IdmServerProxyReadTransaction<'a> {
                     .get_ava_single_iname(Attribute::Name)
                     .map(str::to_string)?;
 
+                let has_image = entry.get_ava_single_image(Attribute::Image).is_some();
+
                 Some(AppLink::Oauth2 {
                     name,
                     display_name,
                     redirect_url,
-                    icon: None,
+                    has_image,
```

Comment:
Is there a reason we couldn't just inline the image data here so that it's ready to render?

Reply:
Applinks would be slower because you'd have to wait to receive all image data. (e.g. `/v1/self/_applinks` but also in-lining it into a template)